Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -5545,8 +5545,9 @@ sub GenerateAttributeGetterTrampolineDefinition
         push(@templateParameters, "CastedThisErrorBehavior::Assert");
     }
     
-    push(@$outputArray, "JSC_DEFINE_CUSTOM_GETTER(${attributeGetterName}, (JSGlobalObject* lexicalGlobalObject, EncodedJSValue thisValue, PropertyName attributeName))\n");
+    push(@$outputArray, "static inline JSC_DEFINE_CUSTOM_GETTER(${attributeGetterName}, (JSGlobalObject* lexicalGlobalObject, EncodedJSValue thisValue, PropertyName attributeName))\n");
```

Comment:
Looks like this change causes the bindings tests to fail, will see if I can get it to build without the `static inline` addition.